playwright/tests/qaParagraph.spec.js

```diff
@@ -0,0 +1,19 @@
+const { test, expect } = require('../utils/next-test')
```

Comment:
you can remove this file, we'll only need playwright tests for actual layout pages

Reply:
Gothca